setup.py

```diff
@@ -64,20 +63,21 @@ def _unavailable(self, exc):
         print(exc)
 
 
-if Feature:
-    # Optional C extension module for speeding up Genshi:
-    # Not activated by default on:
-    # - PyPy (where it harms performance)
-    # - CPython >= 3.3 (the new Unicode C API is not supported yet)
-    speedups = Feature(
-        "optional C speed-enhancements",
-        standard = not is_pypy,
-        ext_modules = [
-            Extension('genshi._speedups', ['genshi/_speedups.c']),
-        ],
-    )
-else:
-    speedups = None
+# Optional C extension module for speeding up Genshi
+speedups = Extension('genshi._speedups', ['genshi/_speedups.c'])
+ext_modules = []
+# Not activated by default on:
+# - PyPy (where it harms performance)
+try:
+    _speedup_enable_default = 0 if is_pypy else 1
```

Comment:
Setting `_speedup_enable_default` could live outside the `try: ...` block.

Reply:
Sure.